platform/ios/PolyPodApp/Features/Feature.swift

```diff
@@ -11,35 +12,57 @@ class Feature {
     let thumbnail: URL?
     private let links: [String: String]
     
-    static func load(path: URL, languageCode: String?) -> Feature? {
-        let manifest = readManifest(path)
+    static func load(path: URL) -> Feature? {
+        guard let manifest = readManifest(path) else {
+            return nil
+        }
         return Feature(
             path: path,
-            manifest: manifest,
-            languageCode: languageCode
+            manifest: manifest
         )
     }
     
-    init(path: URL, manifest: FeatureManifest, languageCode: String?) {
+    init(
+        path: URL,
+        name: String?,
+        author: String?,
+        description: String?,
+        thumbnail: String?,
+        thumbnailColor: String?,
+        primaryColor: String?,
+        links: [String: String]?
+    ) {
         self.path = path
-        let userLanguage = languageCode ?? "en"
-        let translations = manifest.translations?[userLanguage]
         let id = path.lastPathComponent
         self.id = id
-        name = translations?.name ?? manifest.name ?? id
-        author = translations?.author ?? manifest.author
-        description = translations?.description ?? manifest.description
-        let primaryColor = parseColor(hexValue: translations?.primaryColor ?? manifest.primaryColor)
+        self.name = name ?? id
+        self.author = author
+        self.description = description
+        let primaryColor = parseColor(hexValue: primaryColor)
         self.primaryColor = primaryColor
-        thumbnailColor = parseColor(hexValue: translations?.thumbnailColor ?? manifest.thumbnailColor) ?? primaryColor
-        thumbnail = findThumbnail(
+        self.thumbnailColor = parseColor(hexValue: thumbnailColor) ?? primaryColor
```

Comment:
That logic belongs into manifest IMHO - but we could move it later, let's keep this large PR from getting massive :D Makes for a good first bug anyway.

Reply:
Not quite. A specific "problem" with this `Feature` class is that it is SwiftUI specific. I mean `Color` is SwiftUI type. Ideally the `Color` should be derived in SwiftUI View from the FeatureManifest field.

Reply:
The logic I'm referring to is that _if_ there's no `thumbnailColor`, we use the `primaryColor`. There's various ways we could move that logic to the core, one trivial way being to define a `fallbackColor` that's just set to the value of `primaryColor`, but I think we can do better. But like I said, better to address that later.

Reply:
Ok, I assumed that `Color` parsing from hex can fail, but revisiting the code I see that `Color` can be nil only if hex value is nil. Yeah, this logic can be part of FeatureManifest parsing in Core.